bedrock/firefox/templates/firefox/features/safebrowser.html

```diff
@@ -0,0 +1,222 @@
+      </p>
+      <p>
+        {% trans %}
+        That's why we block third-party trackers and offer content blocking on private and regular browsing mode.
+        {% endtrans %}
+      </p>
+    </section>
+
+    <section class="article-section">
+      <h2>{{ _('Private Browsing') }}</h2>
+      <p>
+        {% trans %}
+        Speaking of private browsing, it's a powerful privacy tool that's easy to use. On Firefox, private browsing
+        means we
+        don't save information about where you've been, clicked or looked. Private browsing leaves smaller digital
+        footprints,
+        giving away less about what you do and where you go online.
+        {% endtrans %}
+      </p>
+    </section>
+
+    <section class="article-section">
+      <h2>{{ _('Containers') }}</h2>
+      <p>
+        {% trans %}
+        Cleaning up your on line space isn't just good feng shui, it can lessen the worry of being tracked. Separate
+        identities
+        for your work websites or your personal interests keep trackers confined to one area. Ads that have nothing to
+        do with
+        work won't pop-up in the middle of a presentation. For this purpose, we created Containers.
+        {% endtrans %}
+      </p>
+      <p>
+        {% trans %}
+        Containers help you stay organized and keep trackers in their place. Checking flight prices? Log onto the same
+        website
+        on different containers to make sure the price doesn't mysteriously go up the second time you visit.
+        {% endtrans %}
+      </p>
+      <p>
+        {% trans %}
+        If you're concerned about how much Facebook already knows about you, try the Facebook Container. It makes it
+        harder for
+        Facebook to track your activity on other websites with third-party cookies. So you can keep trading funny memes
+        with
+        friends without Facebook collecting data on your moves.
+        {% endtrans %}
+      </p>
+    </section>
+
+    <section class="article-section">
+      <h2>{{ _('Firefox Focus') }}</h2>
+      <p>
+        {% trans %}
+        Security is about your peace of mind. You shouldn't have to worry about whether you're on a safe browser. Using
+        Firefox
+        Focus means you never have to think about internet privacy on a mobile device. Its privacy features are always
+        on, and
+        your browsing history is wiped clean after each session. No trace, no trackers, no data for hackers to steal.
+        {% endtrans %}
+      </p>
+    </section>
+
+    <section class="article-section">
+      <h2>{{ _('Firefox Monitor') }}</h2>
+      <p>
+        {% trans %}
+        It's hard to go a week without hearing of a new data breach. If you're a human who lives and works, you probably
+        don't
+        have the time to keep track of data news. We thought it might be nice to have a tool that kept watch on accounts
+        and
+        could let you know if anything fishy is going on.
```

Comment:
We should leave an l10n note about the meaning of "fishy" in this context.
https://bedrock.readthedocs.io/en/latest/l10n.html#translating-with-lang-files
```
{# L10n: "Fishy" in this context is an adjective meaning suspicious or questionable. #}
```